### ISSUE TITLE
missing leading `{` when rendering JSON string 

### ISSUE THREAD

**Issue by Zane-XY** [L-Briand/KTM#2]
It seems the rendering logic has a bug when dealing with the following input:

```kotlin
       val prompt = """
            {
              "model": "{{{model}}}",
            }""".trimIndent()
        val p = prompt.render(
            mapOf( "model" to "staticValue1")
        )
        println(p)
```
The result prints: 
```
  "model": "staticValue1",
}
```
The leading `{` is missing in the rendered result.

**Comment by L-Briand** [comment]
Hello @Zane-XY 

Thanks for sharing, I've found the cause. I will patch it ASAP.

**Comment by L-Briand** [comment]
New version `1.0.2` should fix this issue. Can you please try ?
```kotlin
    implementation("net.orandja.ktm:core:1.0.2")
```

**Comment by Zane-XY** [comment]
hi @L-Briand ,

i tried, it works for some cases, but not this one:

```
    @Test
    fun testMapContext() {
        val prompt = """
{ 
  "source":"{{{model}}}"
}
    """.trimIndent()
        val p = prompt.render(
            mapOf( "model" to "staticValue1")
        )
        println(p)
    }
}
```

**Comment by L-Briand** [comment]
@Zane-XY Thank you for the comeback. I'll look into it.

**Comment by L-Briand** [comment]
The issue should be fixed in the 1.0.3 version
